.github/workflows/cache_data.yaml

```diff
@@ -38,13 +38,13 @@ jobs:
       # Install GMT and other required dependencies from conda-forge
       - name: Install dependencies
         run: |
-          mamba install gmt=6.3.0 \
-                        numpy pandas xarray netCDF4 packaging matplotlib
+          mamba install gmt=6.3.0 numpy pandas xarray netCDF4 packaging \
+                        build matplotlib
```

Comment:
A bit unrelated, but why is matplotlib needed in the cache_data workflow? Isn't this GitHub Actions workflow just used to download files?

I see that the `matplotlib` dependency was added at 270e3ea29a72058c277922cc16c2fbf225f63ea1 in #1310, but can't see exactly why this is needed.

Reply:
Ah ok. Seems like that `import` line could just be put inside the `check_figures_equal` function, but let's just keep things the way it is right now.